common/terms.html

```diff
@@ -63,6 +63,12 @@
     <dd>
       <p>A set of instance <a class="termref" href="#dfn-object">objects</a> that share similar characteristics.</p>
     </dd>
+    <dt><dfn>Composite</dfn></dt>
+    <dd>
+      <p>A <a>widget</a> that may contain navigable descendants or <a>owned</a> children.</p>
+      <p>Authors SHOULD ensure that a composite widget exists as a single navigation stop within the larger navigation system of the web page. Once the composite widget has focus, authors SHOULD provide a separate navigation mechanism for users to navigate to <a>elements</a> that are descendants or owned children of the composite element.</p>
```

Comment:
(For some reason, github won't let me create a "suggestion". Draft, maybe?).

I realize that these are the words that were currently in the spec, but they're weirdly confusing. Is there really some other "larger navigation system of the web page" out there (not including AT navigation, because that's not really relevant to the description of a composite)?
Shouldn't we just say "tab order" so as not to confuse authors?

`<p>Authors SHOULD ensure that a composite widget exists as a single tab stop within the tab order of the web page. Once the composite widget has focus, authors SHOULD provide a separate navigation mechanism, such as arrow keys, for users to navigate to <a>elements</a> that are descendants or owned children of the composite element.</p>`